stable/appmesh-spire-agent/templates/service.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "appmesh-spire-agent.fullname" . }}
```

Comment:
namespace as well

Reply:
Mainly to leave it up to the user and also to be consistent with how we do it with other 3rd party appmesh integrations.

https://github.com/aws/eks-charts/blob/master/stable/appmesh-grafana/templates/deployment.yaml#L4      
https://github.com/aws/eks-charts/blob/master/stable/appmesh-prometheus/templates/service.yaml#L4
https://github.com/aws/eks-charts/blob/master/stable/appmesh-jaeger/templates/service.yaml#L4

README.md specifies it exclusively,

helm upgrade -i appmesh-spire-agent eks/appmesh-spire-agent \
--namespace spire